docs/examples/src/main.c

```diff
@@ -0,0 +1,22 @@
+#include "divide.h"
```

Comment:
In the parse output below this example file contains only STYLE results.
- Don't use the --enable-all option for the analyze command because it is not recommended. It will generate to much noise.
- Also, please put an extra error to this example file, what Clang-Tidy will find and not a style checker. And for the analyze/check command enable this checker if it is not enabled by default.
- Don't forget to update the output below if you do the changes.

Reply:
Done.